_data/paras1810/paras1810.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Paras Jain # No longer than 18 characters
+institution: Maharaja Surajmal Of Technology  # no longer than 58 characters
+profile_pic: paras.jpg # Relative path to the image you uploaded
```

Comment:
Incorrect file name, please do one of the following:
1. Change `paras.jpg` to `paras1810.jpg` in the markdown file.
2. Rename `paras1810.jpg` to `paras.jpg`.
I'd suggest you make the change in the markdown file as it matches you GitHub username!